### PR TITLE
Add urgent banner component and styles

### PR DESCRIPTION
**Pull request by BarryGeeMOJ**
### Context

> Does this issue have a Trello card?
https://trello.com/c/Isn6DtTl/997-allow-prisoners-to-see-urgent-notices-in-the-banner?filter=label:Front+End

> If this is an issue, do we have steps to reproduce?
n/a

### Intent

> What changes are introduced by this PR that correspond to the above card?
- Added a new Urgent banner component and supporting styles

> Would this PR benefit from screenshots?
Screenshot 2022-08-30 at 15 32 06
Screenshot 2022-08-30 at 15 32 22

Screenshot 2022-08-31 at 07 22 49

### Considerations

> Is there any additional information that would help when reviewing this PR?
n/a

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: assets/sass/components/_urgent-banner.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  .govuk-notification-banner__link:hover {
+    color: #922615;
+  }
+
+  .govuk-notification-banner__link:active,
+  .govuk-notification-banner__link:focus {
+    color: #000000;
+  }
+
+  @media (max-width: $tablet-breakpoint) {
```
Comment (jonhindle) [inline assets/sass/components/_urgent-banner.scss:27]: For consistency this should probably be:
```@include govuk-media-query($until: tablet) {```

File: server/repositories/cmsQueries/urgentBannerQuery.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  path() {
+    return `/jsonapi/prison/${this.establishmentName}/node/urgent_banner?${this.query}`;
+  }
+
+  transformEach(banner) {
+    return {
+      title: banner.title,
+      more_info_link: banner.fieldMoreInfoPage?.path?.alias,
```
Comment (jonhindle) [inline server/repositories/cmsQueries/urgentBannerQuery.js:28]: camelCase required - `moreInfoLink`

File: server/utils/jsonApi.js [hunk 3]
```diff
@@ -76,6 +76,9 @@ const getCategoryId = categories => {
   };
 };
 
+const isUnpublished = ({ unpublishOn = null }) =>
+  !unpublishOn || unpublishOn >= new Date().getTime();
```
Comment (leonk) [inline server/utils/jsonApi.js:80]: Have we checked how this handles timezones?
The unpublished date should always be in UTC, so as long as `new Date().getTime()` is also UTC we should be OK.

Comment (jonhindle) [reply server/utils/jsonApi.js:80]: getTime return UTC - so this should be fine 👍  Its also all server side.

**Review by leonk — DISMISSED** [review]
Lgtm

**Review by leonk — APPROVED** [review]
LGTM

**Pull request merged**